multipers/tests/test_mma.py

```diff
@@ -14,8 +14,8 @@ def test_1():
     st.insert([1], [1, 0])
     st.insert([0, 1], [1, 1])
     mma_pymodule = st.persistence_approximation()
-    assert np.array_equal(mma_pymodule[0].get_birth_list(), [[1.0, 0.0], [0.0, 1.0]])
-    assert np.array_equal(mma_pymodule[0].get_death_list(), [[np.inf, np.inf]])
+    assert np.array_equal(mma_pymodule[0].get_birth_list(), [[0.0, 1.0], [1.0, 0.0]])
+    assert np.array_equal(mma_pymodule[0].get_death_list(), [[np.inf]])
```

Comment:
I'm not super fan of this... this inf is only a cpp artifact, it shouldn't be shown in the python interface

Reply:
I am not sure to understand ? You don't like that there is an infinity value or that a list at infinity has size 1 and not number of parameters?

Reply:
the output should be `[[inf,inf,...,inf]]` in python indeed. This is confusing otherwise.

Reply:
and prone to bug

Reply:
So, you just want that `PySummand.get_birth/death_list` always returns a list of size "number of parameters" ?